graylog2-web-interface/src/components/navigation/bindings.ts

```diff
@@ -63,6 +63,7 @@ const navigationBindings: PluginExports = {
             { path: Routes.SYSTEM.LOOKUPTABLES.OVERVIEW, description: 'Lookup Tables', permissions: ['lookuptables:read'] },
             { path: Routes.SYSTEM.PIPELINES.OVERVIEW, description: 'Pipelines', permissions: ['pipeline:read', 'pipeline_connection:read'] },
             { path: Routes.SYSTEM.SIDECARS.OVERVIEW, description: 'Sidecars', permissions: ['sidecars:read'] },
+            { path: Routes.SYSTEM.ENTITYGROUPS.OVERVIEW, description: 'Manage Entity Groups' },
```

Comment:
Seems to fit the pattern of the other items here to just have `Entity Groups`, with `Manage` being implied?
```suggestion
            { path: Routes.SYSTEM.ENTITYGROUPS.OVERVIEW, description: 'Entity Groups' },
```

Reply:
@ryan-carroll-graylog Yup, that thought definitely crossed my mind but without any mockups at the moment and with the plan to continue working on this next cycle I decided to leave out the fields to add entities to a group for now.

Reply:
Ah ok definitely makes sense to me to wait for mockups. The API is pretty simple for anyone wanting to use this as is anyway.